projects/Mallard/package.json

```diff
@@ -48,5 +49,19 @@
     },
     "jest": {
         "preset": "react-native"
+    },
+    "workspaces": {
+        "nohoist": [
+            "react-native",
+            "react-native/**",
+            "react-native-zip-archive",
+            "react-native-svg",
+            "react-navigation",
+            "react-navigation/**",
+            "rn-fetch-blob",
+            "rn-fetch-blob/**",
+            "@react-native-community/async-storage",
+            "@guardian/pasteup"
+        ]
```

Comment:
React Native and Yarn Workspaces are [not currently friends. [The workaround](https://github.com/connectdotz/yarn-nohoist-examples) is to prevent Yarn Workspaces hoisting any modules that generate native code. I'm not sure why `pasteup` has to be in there too, it just does. Let me know if you have any questions / suggestions